src/plugins/data/common/index.ts

```diff
@@ -17,12 +17,12 @@
  * under the License.
  */
 
-export * from './query';
+export * from './constants';
+export * from './es_query';
 export * from './field_formats';
-export * from './kbn_field_types';
 export * from './index_patterns';
-export * from './es_query';
-export * from './utils';
```

Comment:
Removed this export because now the only thing in `utils` is internal to other items in the `common` directory.